pandas/core/frame.py

```diff
@@ -924,21 +924,26 @@ def from_dict(cls, data, orient='columns', dtype=None, columns=None):
         return cls(data, index=index, columns=columns, dtype=dtype)
 
     def to_dict(self, orient='dict', into=dict):
-        """Convert DataFrame to dictionary.
+        """
+        Convert DataFrame to dictionary.
+
+        Method converting the DataFrame to a Python dictionary,
+        the type of the key-value pairs can be customized with
+        the parameters (see below).
 
         Parameters
         ----------
         orient : str {'dict', 'list', 'series', 'split', 'records', 'index'}
             Determines the type of the values of the dictionary.
 
-            - dict (default) : dict like {column -> {index -> value}}
-            - list : dict like {column -> [values]}
-            - series : dict like {column -> Series(values)}
-            - split : dict like
-              {index -> [index], columns -> [columns], data -> [values]}
-            - records : list like
+            - 'dict' (default) : dict like {column -> {index -> value}}
+            - 'list' : dict like {column -> [values]}
+            - 'series' : dict like {column -> Series(values)}
+            - 'split' : dict like
+              {index' -> [index], columns -> [columns], data -> [values]}
```

Comment:
Can you on this line do `{'index' -> [index], 'columns' -> [columns], 'data' -> [values]}` (as in this case those are actual strings and not a placeholder for what is in the data)